resources/fetchers/eks_factory.go

```diff
@@ -0,0 +1,62 @@
+package fetchers
+
+import (
+	"github.com/elastic/cloudbeat/resources/providers/awslib"
+
+	"github.com/elastic/beats/v7/libbeat/common"
+	"github.com/elastic/cloudbeat/resources/fetching"
+	"github.com/elastic/cloudbeat/resources/manager"
+)
+
+const (
+	EKSType = "aws-eks"
+)
+
+func init() {
+	manager.Factories.ListFetcherFactory(EKSType, &EKSFactory{
+		extraElements: getEksExtraElements,
```

Comment:
I don't really like this pattern but let's try to think of a better way to load it on the next iterations